app/views/PageParents.vue

```diff
@@ -66,7 +522,29 @@
 
       ...mapActions({
         loadProducts: 'products/loadProducts'
-      })
+      }),
+
+      openPremiumSubscribeModal () {
+        this.subscribeModalOpen = true
+      },
+
+      subscribeBasic () {
+        this.$refs.subscribeModal.$once('shown', () => {
+          const modal = this.$refs.subscribeModal.$data.modalViewInstance
+          modal.onClickPurchaseButton()
+        })
+
+        this.openPremiumSubscribeModal()
+      },
+
+      subscribeLifetime () {
+        this.$refs.subscribeModal.$once('shown', () => {
+          const modal = this.$refs.subscribeModal.$data.modalViewInstance
+          modal.onClickStripeLifetimeButton()
```

Comment:
If I understand this correctly, this method is run on the `PremiumSubscribeModal` triggered on line 546?

The exact path `$refs.subscribeModal.$data.modalViewInstance` is slightly over my head. It looks like we are tunneling into the BackboneModalHarness. Maybe could use a comment.

Reply:
Yeah good call, I'd intended to comment this code (and forgot) because it's a hack to advance the modal until we can move the subscription code out of the modal and into a vuex store.